src/components/modules/MainPage/ShortCard/StyledShortCard.js

```diff
@@ -86,13 +87,15 @@ export const Input = styled.button`
   background-color: rgba(123, 123, 123,0.2)
 `;
 
-export const ProjectContainer = styled.div`
-  display: flex;
-  justify-content: space-between;
-  border-radius:40px;
-  width: 100%;
-  @media (orientation:portrait) {
-    width: 100%;
-    flex-direction: column;
-  }
-`;
\ No newline at end of file
+export const Section = styled.div`
+&:before{
+    content:'';
```

Comment:
This can be used without 
``&:before { content: ''; }``

Reply:
without  content: ' ';   the dividing line despeared